op-chain-ops/genesis/config.go

```diff
@@ -734,6 +734,13 @@ func NewL2StorageConfig(config *DeployConfig, block *types.Block) (state.Storage
 	//	return storage, errors.New("block base fee not set")
 	//}
 
+	var baseFee *big.Int
+	if config.Fermat != nil && config.Fermat.Cmp(big.NewInt(0)) <= 0 {
```

Comment:
```
config.Fermat.Cmp(big.NewInt(0)) <= 0
```
Is there a problem with the code here that seems to determine Fermat as a negative number?

Reply:
There is no problem. Although the negative number check is missing, the initial chain will generally be set to greater than or equal to 0. Other hard forks set according to the block height also use the `big.int` type.